src/bootstrap/defaults/config.tools.toml

```diff
@@ -1,5 +1,7 @@
 # These defaults are meant for contributors to tools which build on the
 # compiler, but do not modify it directly.
+# Using these defaults will download the stage2 compiler (see `download-rustc`
+# setting) and the stage2 toolchain should therefore be used for these defaults.
```

Comment:
Please put this comment directly on download-rustc so it gets copied over when I add it to other profiles (see https://github.com/rust-lang/compiler-team/issues/619). If you're worried people won't see it, you can move download-rustc higher in the file so it's the first option people see.